### DISCUSSION TITLE
unique IDs with multiple node types

### DISCUSSION THREAD

**Discussion by felipemello1** [callahantiff/PheKnowLator#104] [General]
Hi, I was working with pheknowlator and found an error in my processed data. After further investigation, I realized that there are unique IDs that can be identified as DRUG, CATALYST and COFACTOR at the same time. I am not sure how I should handle it:
option 1) Should I have 3 different nodes and fragment the information about the ID;
option 2) Should the ID be a single node with 3 edge type families (DRUG-?, CATALYST-?, COFACTOR-?) ?

I prefer option 2, but in that case, I can't say that the same node is at the same time a DRUG, a CATALYST and a COFACTOR. Option 1 seems easier, but it doesnt sound right to scatter the information about the node.

Any ideas? Thanks in advance!

**Comment by callahantiff** [comment]
Hi @fmellomascarenhas -

Thanks for your comment, and for using PheKnowLator! 😄 🎉 

I want to make sure I fully understand what you mean. Can you please provide some additional information about your use case and what PheKnowLator build you are using (i.e., instance or subclass-baed and the OWL or OWL-NETS version)? I have a few ideas that would be easy to implement that would work for option 2, but I want to make sure that it would not complicate your downstream applications.

**Reply by felipemello1** [reply]
Sure, I will provide it next Monday. Thanks for answering!

**Reply by callahantiff** [reply]
Sounds great!
